test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -156,28 +156,12 @@ bootstrap_remote_freebsd()
         "
 
     if [ "${controller}" ]; then
-        jinja2_pkg="py${python_package_version}-jinja2"
-        cryptography_pkg="py${python_package_version}-cryptography"
-        pyyaml_pkg="py${python_package_version}-yaml"
-
-        # Declare platform/python version combinations which do not have supporting OS packages available.
-        # For these combinations ansible-test will use pip to install the requirements instead.
-        case "${platform_version}/${python_version}" in
-            "12.4/3.9")
-                ;;
-            *)
-                jinja2_pkg=""  # not available
-                cryptography_pkg=""  # not available
-                pyyaml_pkg=""  # not available
-                ;;
-        esac
-
         packages="
             ${packages}
             libyaml
-            ${pyyaml_pkg}
-            ${jinja2_pkg}
-            ${cryptography_pkg}
+            py${python_package_version}-yaml
+            py${python_package_version}-jinja2
+            py${python_package_version}-cryptography
```

Comment:
The only changes that should be made here are to remove the following lines, which are responsible for indicating we should use OS packages for Python 3.9 on FreeBSD 12.4:

https://github.com/ansible/ansible/blob/ed8a404f4a736bba2dc6b32f0c79f8b7e5dabf61/test/lib/ansible_test/_util/target/setup/bootstrap.sh#L166-L167

Those two lines are no longer needed since we're not supporting Python 3.9 as a controller.

The remaining lines should be kept so the correct OS + Python version combinations are supported with OS packages. At the moment, there aren't any (hence the only entry being `*`), but since that usually changes every time we add a new FreeBSD version, it's better to keep the comments and additional logic so we don't have to dig it up later (we just add a new entry to the case statement instead).

Making this change should fix the CI failures.